spec/vorpal/performance/performance_spec.rb

```diff
@@ -100,47 +100,6 @@ def add_branch(branch_options)
     end
   end
 
-  # it 'creates aggregates quickly' do
-  #   trees = build_trees(1000)
-  #
-  #   puts 'starting persistence benchmark'
-  #   puts Benchmark.measure {
-  #     tree_mapper.persist(trees)
-  #   }
-  # end
-  #
-  # it 'updates aggregates quickly' do
-  #   trees = build_trees(1000)
-  #
-  #   tree_mapper.persist(trees)
-  #
-  #   puts 'starting update benchmark'
-  #   puts Benchmark.measure {
-  #     tree_mapper.persist(trees)
-  #   }
-  # end
-  #
-  # it 'loads aggregates quickly' do
-  #   trees = build_trees(1000)
-  #   tree_mapper.persist(trees)
-  #   ids = trees.map(&:id)
-  #
-  #   puts 'starting loading benchmark'
-  #   puts Benchmark.measure {
-  #     tree_mapper.query.where(id: ids).load_many
-  #   }
-  # end
-  #
-  # it 'destroys aggregates quickly' do
-  #   trees = build_trees(1000)
-  #   tree_mapper.persist(trees)
-  #
-  #   puts 'starting destruction benchmark'
-  #   puts Benchmark.measure {
-  #     tree_mapper.destroy(trees)
-  #   }
-  # end
-
```

Comment:
These tests exist because performance issues usually only affect one of Vorpal's main operations. When there is a performance issue, these templates allow you to fine-tune your test for maximum iteration speed without modifying the main performance test, which should never change because it is around to show how Vorpal performance changes over time.